.github/workflows/grid_search.yaml

```diff
@@ -47,7 +47,15 @@ jobs:
               echo "experiments running"
               sleep 30
           done
-          dvc queue status 
+          dvc queue status
+          
+          # show all logs
+          for i in $(dvc queue status | awk '{
+          if ($1 != "" && $1 != "Worker" && $1 != "Task")
+            print $1
+          }'); do 
+            dvc queue logs $i
```

Comment:
@dacbd I have merged your changes into the gridsearch branch which triggers the pipeline to run. This time the entire pipeline failed with `ERROR: No output logs found for experiment '51f78a7'`, see: https://github.com/gromag/GridSearchTest/actions/runs/3496300724/jobs/5854051066